README.md

```diff
@@ -28,3 +28,31 @@ We hope the above features make this a tool that developers can use to speed up
     * ASP.NET and web development
     * [Latest .NET Core cross-platform development](https://www.microsoft.com/net/)
     * [Latest .NET SDK](https://www.microsoft.com/en-us/download/details.aspx?id=19988)
+
+### Creating an Instance of the Template (Your Own Blog)
+First, you'll need to get the [newest templating code](https://github.com/dotnet/templating).
+Switch the Branch to "rel/2.0.0-servicing." Then click the green "Clone or download" button. Copy the link in the dropdown that appears.

```

Comment:
I'll just keep this when in doubt, but if we find that it's confusing, then we can always delete it in the future. Good catch though!